Gemfile.lock

```diff
@@ -99,4 +99,4 @@ DEPENDENCIES
   webrick (~> 1.7)
 
 BUNDLED WITH
-   2.4.7
+   2.4.7
```

Comment:
Revert changes to this file, Levi had issues with that. It should be in the gitignore, so not sure what happened here.

Reply:
Apparently the `.gitignore` does not work with `Gemfile.lock` because it was already committed? In any case, here, there is a "silent" change that deletes the newline character at the end of the file. This will revert the change to main:
```
git checkout main -- Gemfile.lock
```
Make sure you have pulled on `main` before doing this:
```
git checkout main
git pull
git checkout feature/publication-tab
git checkout main -- Gemfile.lock
```